openwisp_notifications/static/openwisp-notifications/js/notifications.js

```diff
@@ -238,7 +238,14 @@ function notificationWidget($) {
         }
         klass = notificationReadStatus.get(elem.id);
 
-        // Remove hyperlinks from notification messages if description is present
+        // If description is present, remove hyperlinks from the message
+        let message;
+        if (elem.description) {
+            message = elem.message.replace(/<a [^>]*>([^<]*)<\/a>/g, '$1');
```

Comment:
The comment should be above this line, I would write something like `Remove hyperlinks from generic notifications to enforce the opening of the message dialog`.
This summarizes the work we've been doing and will help us remember why we did this even 2-3 years from now.
It's also useful for new developers who are studying the codebase.